### PR TITLE
Stack version 2020070100

### PR DESCRIPTION
**Pull request by garthwilliamson**
**Describe your changes and to whom it might be helpful**

Hi,

 I added a new version. It makes more tests pass with the latest moodle stack, but not all of them, so I don't know if you want to merge it. Let me know if you want any changes, or feel free to cherry-pick and edit.

It looks like assets/optimize.mac no longer needs to be updated. Is this true? If so, I could update the README to reflect that.

Cheers,
Garth

**Checklist**
- [x] [Applied best practice writing docker files](https://developers.redhat.com/blog/2016/02/24/10-things-to-avoid-in-docker-containers/)
- [x] README.md updated or doen't require changes
- [ ] `assets/maximalocal.mac.template` and `assets/optimize.mac` still match the STACK-version
  _Usually the LMS (ILIAS/ Moodle) plugins create a file in which they store their settings for STACK-Maxima, for example [defining a default size for plots](https://github.com/uni-halle/maximapool-docker/blob/develop/assets/maximalocal.mac.template#L21) or defining which modules to load._
  _In ILIAS the file is located under `/data_dir/xqcas/stack/maximalocal.mac`, in Moodle `$MOODLEDATA/stack/**/maximalocal.mac`._
  _This settings file is then loaded into any non-optimized STACK-Maxima. If optimization is performed, the settings file has to be pre-generated and is baked into the optimized STACK-Maxima._
- [x] No code smells introduced (e.g. used ShellCheck for Shell scripts)

### CODE REVIEW

File: assets/maximalocal.mac.template [hunk 1]
```diff
@@ -29,12 +29,12 @@ STACK_SETUP(ex):=block(
...
-    stack_unit_other_unit_code:[min, amu, u, mmHg, bar, cc, gal, mbar, atm, torr, rev, deg, rpm, K, day, year, in, ft, mi],
-    stack_unit_other_unit_conversions:[s*60, amu, amu, 133.322387415*Pa, 10^5*Pa, m^3*10^(-6), 3.785*l, 10^2*Pa, 101325*Pa, 101325/760*Pa, 2*pi*rad, pi*rad/180, pi*rad/(30*s), K, 86400*s, 3.156e7*s, in, 12*in, 5280*12*in],
-    stack_unit_other_unit_tex:["\\mathrm{min}", "\\mathrm{amu}", "\\mathrm{u}", "\\mathrm{mmHg}", "\\mathrm{bar}", "\\mathrm{cc}", "\\mathrm{gal}", "\\mathrm{mbar}", "\\mathrm{atm}", "\\mathrm{torr}", "\\mathrm{rev}", "\\mathrm{{}^{o}}", "\\mathrm{rpm}", "\\mathrm{K}", "\\mathrm{day}", "\\mathrm{year}", "\\mathrm{in}", "\\mathrm{ft}", "\\mathrm{mi}"],
+    stack_unit_si_unit_code:[m, l, L, g, t, s, h, Hz, Bq, cd, N, Pa, cal, Cal, Btu, eV, J, W, A, ohm, C, V, F, S, Wb, T, H, Gy, rem, Sv, lx, lm, mol, M, kat, rad, sr, K, VA, eV],
```
Comment (Rillke) [inline assets/maximalocal.mac.template:32]: Are these additional units specific to your installation? Are they important for making tests pass? If so, where are the tests defined?

**Review by Rillke — COMMENTED** [review]
Thanks for the PR. There is a question regarding the addition of units.

File: assets/maximalocal.mac.template (hunk 1 above)
Comment (garthwilliamson) [reply assets/maximalocal.mac.template:32]: These units were generated and inserted into stack/**/maximalocal.mac.template, I'm unsure if they are actually needed or not for the tests, I can investigate further if that'll help? The changes on the Moodle side were too extensive for me to quickly understand what was changed there.

Comment (Rillke) [reply assets/maximalocal.mac.template:32]: Units were added in https://github.com/maths/moodle-qtype_stack/commit/0a1683ed9fb2ab1d78bdef279f4e2d90d3bb4a28 and https://github.com/maths/moodle-qtype_stack/commit/9013187ade326c2213e0a31c5ce318335813acd4

**Pull request merged**